omnibus.rb

```diff
@@ -44,9 +44,14 @@
 fetcher_retries 3
 fetcher_read_timeout 120
 
-# We limit this to 10 workers to eliminate transient timing issues in the
-# way Ruby (and other components) compiles on some more esoteric *nixes.
-workers 10
+# Work around for solaris zones, provide a less-bad number of workers
+if solaris? && Ohai['cpu']['total'] == 0
+  workers 10
+end
```

Comment:
We don't overwrite, we could modify the build flags (like how we do the timestamp flag on a release) on the Solaris nodes if needed.

End result is omnibus will try and give all the workers (like 255 on our specific setup) so we hard coded it to 10 (remember that thing we ran into with the sparc builders taking like 5 hours?). The other side note was that we ran into timing issues (miniruby threads building ruby running into each other) when we allowed it to have > 10

I believe the viable options would be:
a) inject 'workers = 10' on our solaris nodes from our zones